employee-rostering-frontend/src/ui/header/Header.tsx

```diff
@@ -18,13 +18,23 @@ import { Brand, PageHeader } from '@patternfly/react-core';
 import * as React from 'react';
 import Navigation from './Navigation';
 import Toolbar from './Toolbar';
+import { useMediaQuery } from 'react-responsive';
 
-const Header: React.FC = () => (
-  <PageHeader
-    logo={<Brand src="/assets/images/optaPlannerLogo200px.png" alt="OptaPlanner Logo" />}
-    toolbar={<Toolbar />}
-    topNav={<Navigation />}
-  />
-);
+const Header: React.FC<{ onNavToggle: () => void} > = ({ onNavToggle }) => {
+  const laptopOrBigger = useMediaQuery({ minWidth: 1400 });
+  return (
+    <PageHeader
+      logo={(
+        <a href="https://www.optaplanner.org/">
```

Comment:
I would prefer if the link opened in a new browser tab. I think if you take most web apps, for example OpenShift web console or Gmail, the top-left corner logo usually takes you to the landing page of the app. I don't have a problem with pointing to www.optaplanner.org but as a user, I'd find it annoying that the link would make me _leave_ the app I'm working with (think about URLs in Gmail message - they always open a new tab).